src/smartcard.rs

```diff
@@ -172,109 +171,9 @@ impl SmartCard {
                 Ok(signature)
             }
             SmartCardApi::PivSmartCard(ref mut scard) => {
-                // select the PIV app
```

Comment:
I decided to remove all APDU command creation. We use emulated smart cards inside sspi, so there is no point in creating raw APDU. We can directly communicate with the smart card